airflow/executors/celery_executor.py

```diff
@@ -34,11 +35,24 @@
 DEFAULT_QUEUE = configuration.get('celery', 'DEFAULT_QUEUE')
 
 
+def getdict(section, key, default=None, deserialize_json=True):
+    if default is None:
+        default = dict()
+
+    try:
+        var = configuration.get(section, key)
+    except AirflowException:
+        return default
+
+    return json.loads(var) if deserialize_json else var
```

Comment:
How can we override worker name with this setup since all of our workers use the same config file?